internal/controllers/module_nmc_reconciler.go

```diff
@@ -302,6 +321,7 @@ func (mnr *ModuleNMCReconciler) SetupWithManager(mgr ctrl.Manager) error {
 	return ctrl.
 		NewControllerManagedBy(mgr).
 		For(&kmmv1beta1.Module{}).
+		Owns(&v1.ConfigMap{}).
```

Comment:
why do we need to watch ConfigMaps?

Reply:
We own them, and we want to reconcile them in case they were altered or deleted.

Reply:
But we want to reconcile them only in case something changed in Module/Node/NMC. In case ConfigMap has changed, what do we care? It won't affect NMC in anyway, no?

Reply:
Moved to a dedicated controller.